lib/Qi/Controller/Assets.pm

```diff
@@ -0,0 +1,44 @@
+package Qi::Controller::Assets;
+use Mojo::Base 'Mojolicious::Controller';
+use Qi::Model::Base;
+use SQL::Abstract;
```

Comment:
No need for this in the Controller.  The purpose of SQL::Abstract is for your model, so it belongs in the model only.